<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medCare</class>
 <widget class="QMainWindow" name="medCare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-image: url(:/images/images/firstfon.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>700</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>650</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(177,202,170);
border-radius:20px;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="page">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>70</y>
             <width>251</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Register in medCare&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="registration_lineEditName">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>150</y>
             <width>281</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
padding:5px 3px;
border-radius:5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Имя</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="registration_lineEditSurname">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>220</y>
             <width>281</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
padding:5px 3px;
border-radius:5px;</string>
           </property>
           <property name="placeholderText">
            <string>Фамилия</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="registration_lineEditMail">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>290</y>
             <width>281</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
padding:5px 3px;
border-radius:5px;</string>
           </property>
           <property name="placeholderText">
            <string>Почта</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="registration_lineEditPassword">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>360</y>
             <width>281</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
padding:5px 3px;
border-radius:5px;</string>
           </property>
           <property name="placeholderText">
            <string>Пароль</string>
           </property>
          </widget>
          <widget class="QPushButton" name="registration_registrationButton">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>430</y>
             <width>201</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
border-radius:5px;</string>
           </property>
           <property name="text">
            <string>Зарегистрироваться</string>
           </property>
          </widget>
          <widget class="QPushButton" name="registration_backwardButton">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>10</y>
             <width>121</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
border-radius:25px;</string>
           </property>
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
          <widget class="QLabel" name="registration_errorLabel">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>520</y>
             <width>401</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qtimages/photo.qrc"/>
 </resources>
 <connections/>
</ui>
